app/assets/javascripts/discourse/tests/integration/components/text-field-test.js

```diff
@@ -24,34 +24,12 @@ module("Integration | Component | text-field", function (hooks) {
     assert.strictEqual(query("input").placeholder, "placeholder.i18n.key");
   });
 
-  test("sets the dir attribute to ltr for Hebrew text", async function (assert) {
+  test("sets the dir attribute to auto when mixed text direction enabled", async function (assert) {
     this.siteSettings.support_mixed_text_direction = true;
 
     await render(hbs`<TextField @value="זהו שם עברי עם מקום עברי" />`);
 
-    assert.strictEqual(query("input").getAttribute("dir"), "rtl");
-  });
-
-  test("sets the dir attribute to ltr for English text", async function (assert) {
-    this.siteSettings.support_mixed_text_direction = true;
-
-    await render(hbs`<TextField @value="This is a ltr title" />`);
-
-    assert.strictEqual(query("input").getAttribute("dir"), "ltr");
-  });
-
-  test("updates the dir attribute when value changes", async function (assert) {
-    this.siteSettings.support_mixed_text_direction = true;
-
-    await render(
-      hbs`<TextField id="mytextfield" @value="This is a ltr title" />`
-    );
-
-    assert.strictEqual(query("input").getAttribute("dir"), "ltr");
-
-    await fillIn("#mytextfield", "זהו שם עברי עם מקום עברי");
-
-    assert.strictEqual(query("input").getAttribute("dir"), "rtl");
+    assert.strictEqual(query("input").getAttribute("dir"), "auto");
```

Comment:
qunit-dom all the things! 😛

```suggestion
    assert.dom("input").hasAttribute("dir", "auto");
```